src/domain/event/EventUtils.ts

```diff
@@ -64,6 +64,24 @@ export const getEventIdFromUrl = (url: string): string | null => {
   return eventId?.[1].replace('/', '') || null;
 };
 
+/*
+ * Format string to price format (add €) if it is a number and is missing currency
+ * For example:
+ * 'random text' -> 'random text'
+ * '2' -> '2 €'
+ * '2.5' -> '2.5 €'
+ * '30/50' -> '30/50 €'
+ * '30-50' -> '30-50 €'
+ */
+export const formatPrice = (price?: string): string => {
+  if (!price) {
+    return '';
+  }
+
+  const priceRegex = /^\d+([/\-.,]{1})?\d+?$/;
+  return price.match(priceRegex) ? `${price} €` : price;
```

Comment:
Open to hear other ideas how to achieve this with regex. Now many price infos are coming without euro sign.

Reply:
In my opinion that looks good enough, even that regex is always hard to understand at first sight.

You could check with something like endsWith, but it's quite cryptic also..:
```
a += !a.endsWith("€") ? " €" : ""
b = !b.endsWith("€") || b + " €"
if (!c.endsWith("€")) c += " €"
```

Reply:
We need to also know if its number and allow / - , . in the middle :) Because regex is hard to understand I added comments to the function definition. But I could also add comment to the regex itself

Reply:
and of course no need to add € if it is just a description. We don't know exactly what is coming from linkedevents :)

Reply:
I tried it here: https://regex101.com/
Tthe site complaisn regex being broken: `/ An unescaped delimiter must be escaped with a backslash (\)`
So perhaps `^\d+([\/\-.,]{1})?\d+?$` is correct?
Also if it has only one number it does not work: "2" -> "2". 
Perhaps more unit tests are needed ;)

Reply:
well actually I removed those because eslint complained and it was still working :D

Reply:
<img width="525" alt="Screenshot 2021-02-24 at 19 01 21" src="https://user-images.githubusercontent.com/15219142/109036859-b82b7e80-76d2-11eb-9bf1-9e8286ce53bd.png">
If you change the flavor to ecmascript it doesn't  complain

Reply:
Anyway, fixed and more tests added